### PR TITLE
Fix builder image for 3.0

### PR DESCRIPTION
**Pull request by jwendell**
It's going to be used only for proxy, as istio and operator use the upstream builder image.

### CODE REVIEW

File: docker/maistra-builder_3.0.Dockerfile [hunk 1]
```diff
@@ -253,9 +122,14 @@ RUN chmod 777 /go && \
     chmod 777 /home/.helm && \
     chmod 777 /home/.gsutil
 
+RUN mkdir -p /work && chmod 777 /work
+WORKDIR /work
+
+ENV PATH=/usr/lib/llvm/bin:/usr/local/google-cloud-sdk/bin:$PATH
```
Comment (yxun) [inline docker/maistra-builder_3.0.Dockerfile:128]: Do we really need `GCLOUD_VERSION` and google-cloud-sdk envs in this builder image ?

**Review by yxun — APPROVED** [review]

File: docker/maistra-builder_3.0.Dockerfile (hunk 1 above)
Comment (jwendell) [reply docker/maistra-builder_3.0.Dockerfile:128]: yes, it's going to be used in post-submit to upload the binary to GCS.

**Pull request merged**